lib/screens/UsersProfileScreen.dart

```diff
@@ -0,0 +1,1576 @@
+                              textAlign: TextAlign.left,
+                              textInputAction: TextInputAction.next,
+                              onChanged: (password) {
+                                if (password.isEmpty) {
+                                  setState(() => unfilledPassword = true);
+                                }
+                                else {
+                                  setState(() => unfilledPassword = false);
+                                }
+                              },
+                            ),
+                          ),
+                          Container(
+                            width: 320,
+                            padding: const EdgeInsets.symmetric(
+                                vertical: 5, horizontal: 5),
+                            decoration: const BoxDecoration(
+                              color: mainScheme,
+                              borderRadius:
+                              BorderRadius.all(Radius.circular(10)),
+                            ),
+                            child: TextField(
+                              maxLines: 1,
+                              controller: _confirmPassword,
+                              obscureText: true,
+                              decoration: unfilledConfirmPassword
+                                  ? globalDecoration.copyWith(
+                                  enabledBorder: const OutlineInputBorder(
+                                      borderSide:
+                                      BorderSide(color: Colors.red)),
+                                  suffixIcon: const Icon(Icons.clear,
+                                      color: Colors.red),
+                                  hintText: 'Confirm Password')
+                                  : globalDecoration.copyWith(hintText: 'Confirm Password'),
+                              style: const TextStyle(
+                                fontSize: 20,
+                                color: Colors.black,
+                              ),
+                              textAlign: TextAlign.left,
+                              textInputAction: TextInputAction.done,
+                              onChanged: (password) {
+                                if (password.isEmpty) {
+                                  setState(() => unfilledConfirmPassword = true);
+                                }
+                                else {
+                                  if (_password.value.text.isEmpty) {
+                                    errorMessage = 'Passwords must match!';
+                                  }
+                                  else if (passwordsMatch()) {
+                                    errorMessage = '';
+                                    unfilledConfirmPassword = false;
+                                  }
+                                  setState(() {});
+                                }
```

Comment:
Elsewhere, you have a validation function for this - I think you can do the same here